test/ventilation-test.h

```diff
@@ -0,0 +1,8 @@
+/*
+ * ventilation-test.h
+ *
+ */
+
+void startVentilation_openInspiratoryValve();
```

Comment:
Consider adding a description for this test case. Maybe prefix it with something that we could scrape out easily in the future to generate a proper test specification... some like: 

```c
/**
 * @test the inspiratory valve opens when ventilation starts
 */
```

Reply:
The name of the function was redefine to be more precise.
I want to reduce the amount of uncessary comments in the code which should be self documented